doc/getting-started.md

```diff
@@ -1,6 +1,11 @@
 Getting started
 ===============
 
+Compiler support
+----------------
+
+Officially supported compilers for production use are the latest releases of _gcc 11_, _gcc 12_ and _gcc 13_.
```

Comment:
Don't love the word official here since it's a community project. Something like "The Firedancer development team tests and supports the following compilers"